overlord/devicestate/devicestate_remodel_test.go

```diff
@@ -952,15 +953,15 @@ func (s *deviceMgrRemodelSuite) TestRemodelClashInProgress(c *C) {
 		"kernel":         "pc-kernel",
 		"gadget":         "pc",
 		"base":           "core18",
-		"required-snaps": []interface{}{"new-required-snap-1", "new-required-snap-2"},
+		"required-snaps": []interface{}{"new-required-snap-1"},
 		"revision":       "1",
 	})
 
 	_, err := devicestate.Remodel(s.state, new)
 	c.Check(err, DeepEquals, &snapstate.ChangeConflictError{
 		Message:    "cannot start remodel, clashing with concurrent one",
 		ChangeKind: "remodel",
-		ChangeID:   "1",
+		ChangeID:   chg.ID(),
```

Comment:
Isn't this fix alone enough to fix flakiness (without changes to required-snaps)?